app/lib/theme/widgets/ReadOnlyCalculatedValueWidget.tsx

```diff
@@ -11,16 +11,19 @@ const ReadOnlyCalculatedValueWidget: React.FC<WidgetProps> = ({
   const isMoney = uiSchema?.isMoney;
   const isPercent = uiSchema?.isPercent;
 
+  const calculatedValue =
+    formContext[uiSchema.calculatedValueFormContextProperty];
+
   return (
     <dd>
-      {formContext.calculatedValue ? (
+      {calculatedValue ? (
```

Comment:
Are we displaying `0` if there is no calculated value? How can users differentiate a `0` from a `null` value?

Reply:
Good catch, we probably want to leave that up to the user:
- only display the calculated value if one has been provided by the user
- allow falsy values to be entered as a calculated values

I'll make a quick update

Reply:
Looping @suhafa into this; we talked about using 0 for the emissions performance calculations specifically